whois-commons/src/main/java/net/ripe/db/whois/common/MaintenanceMode.java

```diff
@@ -35,7 +35,9 @@ public void set(String mode) {
             String[] modes = StringUtils.split(mode, ',');
             world = AccessType.valueOf(modes[0].trim());
             trusted = AccessType.valueOf(modes[1].trim());
-        } catch (RuntimeException e) {}
+        } catch (RuntimeException e) {
+            LOGGER.error("RuntimeException ", e);
```

Comment:
How about LOGGER.error(e.getMessage(), e) ?
The exception class may not be RuntimeException